app/models/training.rb

```diff
@@ -1,4 +1,27 @@
 class Training < ApplicationRecord
   belongs_to :user
   belongs_to :discipline
+
+  def self.from_search(params)
+    params.strip.downcase!
+    to_return = (training_name_match(params) + training_date_match(params))
+    return nil unless to_return
```

Comment:
we don't need it here, if `to_return` is nil it will return this `nil` anyway

Reply:
done